src/components/WikiResults.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+
+import Spinner from './Spinner';
+import ErrorMessage from './ErrorMessage';
+
+import Map from './Map';
+
+const WikiResults = () => {
+  const [data, setData] = useState([]);
+  const [status, setStatus] = useState('idle');
```

Comment:
I'm curious, what does the 'idle' word mean 😄? and great your options for status (loading, success, and error)

Reply:
@alxmcr I can double check with Katia but it's a `loading` state so we could change the wording to be more clear.  Thank you!

Reply:
hey @alxmcr, we have an element of state to know the status of the search.  ``idle`` here means we haven't started doing the search yet.😄

Reply:
Thanks, I'm a little confused but today I learned: `idle` means 'not yet'`.

- _(A little suggestion - no pressure_ 😄) What do you think about `''` (empty string) or 'pending'?

Reply:
hey @alxmcr 😄  not sure ``pending`` would work here. ``pending`` is synonymous with  ``loading``  which is another element of state, used in the code, to indicate we're in the process of loading search results